### PR TITLE
Merge main into dev

### PR DESCRIPTION
**Pull request by mattanglin**
# READY FOR REVIEW

# Summary
- Merging changes in main back into dev. PR for visibility of merge conflicts

# Review By (Date)
- asap

### CODE REVIEW

**Comment by mattanglin** [comment]
@yvonnetangsu Just wanted to get your eyes on this for verification before merging. Most of the conflicts were just quote related and easily resolved by taking latest. There were some changes around `getImageWdith` => `getImageSize` that I resolved by taking `getImageSize`. Additionally the tailwind screen config changes and `perkCardView` composing the `GradientCard`

**Comment by mattanglin** [comment]
Looking into the failing build currently...

**Comment by yvonnetangsu** [comment]
> @yvonnetangsu Just wanted to get your eyes on this for verification before merging. Most of the conflicts were just quote related and easily resolved by taking latest. There were some changes around `getImageWdith` => `getImageSize` that I resolved by taking `getImageSize`. Additionally the tailwind screen config changes and `perkCardView` composing the `GradientCard`

@mattanglin TW screen config changes look good. I expected some nasty conflict in the perk card, and probably a small one in the masthead since that global header PR that got merged yesterday pulled out some of the styling. I can open another PR to manually address those changes later on.

I'll do another look after you resolve build issue. Thanks for doing this!

File: src/components/media/fullWidthImage.js [hunk 1]
```diff
@@ -23,26 +23,24 @@ const FullWidthImage = ({
...
+  let imgAspectRatio = '';
 
-  if (filename != null) {
-    let imgWidth = '';
-
-    // Get image width from URL of storyblok image
-    if (filename?.startsWith('http')) {
-      imgWidth = getImageWidth(filename);
-    }
+  if (filename) {
```
Comment (mattanglin) [inline src/components/media/fullWidthImage.js:28]: Had to fix these image checks since filename is never `null`. Only `undefined` or `''`.

**Comment by mattanglin** [comment]
> > @yvonnetangsu Just wanted to get your eyes on this for verification before merging. Most of the conflicts were just quote related and easily resolved by taking latest. There were some changes around `getImageWdith` => `getImageSize` that I resolved by taking `getImageSize`. Additionally the tailwind screen config changes and `perkCardView` composing the `GradientCard`
> 
> @mattanglin TW screen config changes look good. I expected some nasty conflict in the perk card, and probably a small one in the masthead since that global header PR that got merged yesterday pulled out some of the styling. I can open another PR to manually address those changes later on.
> 
> I'll do another look after you resolve build issue. Thanks for doing this!
> 
> The image changes look good to me also.

@yvonnetangsu This should be good for your eyes again whenever you're free! Will be nice to have Homesite release so we can get these changes into `main` finally.

**Comment by yvonnetangsu** [comment]
> > > @yvonnetangsu Just wanted to get your eyes on this for verification before merging. Most of the conflicts were just quote related and easily resolved by taking latest. There were some changes around `getImageWdith` => `getImageSize` that I resolved by taking `getImageSize`. Additionally the tailwind screen config changes and `perkCardView` composing the `GradientCard`
> > 
> > 
> > @mattanglin TW screen config changes look good. I expected some nasty conflict in the perk card, and probably a small one in the masthead since that global header PR that got merged yesterday pulled out some of the styling. I can open another PR to manually address those changes later on.
> > I'll do another look after you resolve build issue. Thanks for doing this!
> > The image changes look good to me also.
> 
> @yvonnetangsu This should be good for your eyes again whenever you're free! Will be nice to have Homesite release so we can get these changes into `main` finally.

Thank you, @mattanglin! I'm on it 😄

**Review by yvonnetangsu — APPROVED** [review]
LGTM - thank you so much, @mattanglin !

**Pull request merged**